src/main/kotlin/no/nb/bikube/api/core/controller/SwaggerRedirectController.kt

```diff
@@ -0,0 +1,12 @@
+package no.nb.bikube.api.core.controller
+
+import org.springframework.stereotype.Controller
+import org.springframework.web.bind.annotation.GetMapping
+
+@Controller
+class SwaggerRedirectController {
+
+    @GetMapping("/")
+    fun root(): String = "redirect:/swagger-ui/index.html"
```

Comment:
Er det tilstrekkelig?
Ser for meg vi egentli skulle hatt en ResponseEntity med HttpStatus 301 Moved Permanently og noe fancy i location header